app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -131,6 +129,8 @@ public void onCreatePreferences(final Bundle savedInstanceState, final String ro
         newpipeSettings = new File(homeDir + "/databases/newpipe.settings");
         newpipeSettings.delete();
 
+        manager = new ContentSettingsManager(homeDir);
```

Comment:
That causes the build to fail. Can you fix that, please.

Reply:
Oh, how is this possible? I ran the tests successfully on my machine...

Reply:
WOW, i see where this is coming from. PR coming